### PR TITLE
Use good certificate

### LINKED ISSUE #146: Fix Page.goTo tests with BadSSL
#110 

Fix the tests
- `ShouldFailWhenNavigatingToBadSSL`
- `ShouldFailWhenNavigatingToBadSSLAfterRedirects`

### PR DESCRIPTION
**Pull request by kblok**
closes #146

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 1.0.{build}
+branches:
+  only:
+  - master
+image: Previous Visual Studio 2017
+configuration: Release
+before_build:
+- ps: >-
+    dotnet restore .\lib\PuppeteerSharp.sln
```
Comment (Meir017) [inline appveyor.yml]: I think it should be
```yml
ps:
  - dotnet restore .\lib\PuppeteerSharp.sln
  - .\appveyor\installcert.ps1
```

Comment (kblok) [reply appveyor.yml]: Ah, you smart guy

Comment (Meir017) [reply appveyor.yml]: yaml supports multiline strings...

**Comment by Meir017** [comment]
try using https://ci.appveyor.com/tools/validate-yaml
got this:
```yml
version: 1.0.{build}
branches:
  only:
  - master
image: Previous Visual Studio 2017
configuration: Release
before_build:
- ps:
    dotnet restore .\lib\PuppeteerSharp.sln;
    .\appveyor\installcert.ps1
build:
  project: .\lib\PuppeteerSharp.sln
  publish_nuget: true
  include_nuget_references: true
  verbosity: minimal
test_script:
- cmd: >-
    cd .\lib\PuppeteerSharp.Tests;
    dotnet test -s test.runsettings
```

**Comment by Meir017** [comment]
notice the `;` at the end of lines

**Comment by kblok** [comment]
@Meir017 I think now the certificate is too good :joy:

**Comment by Meir017** [comment]
now the tests are failing 😢 
- PuppeteerSharp.Tests.Page.GotoTests.ShouldFailWhenNavigatingToBadSSL
- PuppeteerSharp.Tests.Page.GotoTests.ShouldFailWhenNavigatingToBadSSLAfterRedirects

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,8 +18,10 @@ The test server requires a self-signed certificate at this location: `.\lib\Pupp
 You can run this PowerShell script on Windows
 
 ```
-New-SelfSignedCertificate -Subject "puppeteer" -FriendlyName "Puppeteer" -CertStoreLocation "cert:\LocalMachine\My"
-Get-ChildItem -Path cert:\LocalMachine\my | where { $_.subject -eq "CN=puppeteer" } | Export-Certificate -FilePath .\lib\PuppeteerSharp.TestServer\testCert.cer
+New-SelfSignedCertificate -Subject "localhost" -FriendlyName "Puppeteer" -CertStoreLocation "cert:\CurrentUser\My"
```
Comment (kblok) [inline CONTRIBUTING.md]: The key here was the domain (subject) `localhost` and the store `CurrentUser`

**Comment by kblok** [comment]
Ready to review @Meir017  🚀

**Pull request merged**